strawberryfields/gbs/subgraph.py

```diff
@@ -76,13 +82,12 @@
 
 import networkx as nx
 
-from strawberryfields.gbs import g_sample, utils
+from strawberryfields.gbs import g_sample
```

Comment:
I might be getting confused, but has `g_sample` not been merged as of this PR?

Reply:
Nope it's coming out in the semi-orthogonal PR https://github.com/XanaduAI/strawberryfields/pull/189

The merging isn't so bad for these PR's, there are conflicts but it mainly involves sorting out the imports. How about I only ask you to look at this once 189 gets merged in?